public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx

```diff
@@ -59,12 +61,28 @@ export const PanelHeader: FC<Props> = ({ panel, error, isViewing, isEditing, dat
                   />
                 ) : null}
                 <h2 className={styles.titleText}>{title}</h2>
-                <Icon name="angle-down" className="panel-menu-toggle" />
-                <PanelHeaderMenuWrapper panel={panel} dashboard={dashboard} show={panelMenuOpen} onClose={closeMenu} />
-                {data.request && data.request.timeInfo && (
-                  <span className="panel-time-info">
-                    <Icon name="clock-nine" size="sm" /> {data.request.timeInfo}
-                  </span>
+                {!getConfig().isPublicDashboardView ? (
```

Comment:
```suggestion
                {dashboard.meta.isPublic ? (
```

Reply:
`dashboard.meta.isPublic` only means the dashboard is **allowed** to have public dashboards created from it. The check in `getConfig().isPublicDashboardView()` means this is a public dashboard being viewed publicly.

Reply:
That is not what I mean by this flag, I mean by this flag that the current dashboard is opened using a public url / key

Reply:
The meta flags are for non persisted flags and permissions

Reply:
Ah I see. I thought it was populated by the database `isPublic` flag. I have updated this now